lib/rules/sync-tag.js

```diff
@@ -1,9 +1,8 @@
 const path = require("path");
-const ancesdir = require("ancesdir");
 
 const util = require("../util.js");
 
-const pkgRoot = ancesdir(__dirname, ".pkg_root");
```

Comment:
You could have made that a configurable value, or left it blank so that it would go to the nearest `package.json`.